condition/condition_plugins/plugins.xml

```diff
@@ -49,6 +49,6 @@
   <class name="condition_plugins/SpeedCondition"
          type="condition_plugins::SpeedCondition"
          base_class_type="scenario_conditions::ConditionBase">
-    <description>This is a add plugin.</description>
```

Comment:
@hansrobo
> This is a add plugin.

May I ask what is this mean?

Reply:
cc. @hakuturu583 @yamacir-kit